server/express/package.json

```diff
@@ -20,7 +20,7 @@
     "express": "^4.17.1",
     "morgan": "^1.10.0",
     "passport": "^0.4.1",
-    "passport-did-auth": "^2.0.0-alpha.22",
+    "passport-did-auth": "^2.0.0-alpha.29",
```

Comment:
I merged https://github.com/energywebfoundation/iam-client-examples/pull/49 so best to perhaps undo the changes to `express/package-lock.json` and `express/package.json` and rebase